E2E-tests/src/cardano_cli.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, config: MainChainConfig, cardano_cli: Tool):
         self.run_command = RunnerFactory.get_runner(cardano_cli.ssh, cardano_cli.shell)
 
     def query_tip(self) -> int:
-        cmd = f"{self.cli} query tip {self.network}"
+        cmd = f"{self.cli} latest query tip {self.network}"
```

Comment:
Why is this here? We don't need it with 10.1.1.0 cli.

Reply:
It currently breaks without this. I'd like to fix this in a separate PR.